project_euler/problem_075/sol1.py

```diff
@@ -0,0 +1,53 @@
+"""
+Project Euler Problem 75: https://projecteuler.net/problem=75
+
+It turns out that 12 cm is the smallest length of wire that can be bent to form an
+integer sided right angle triangle in exactly one way, but there are many more examples.
+
+12 cm: (3,4,5)
+24 cm: (6,8,10)
+30 cm: (5,12,13)
+36 cm: (9,12,15)
+40 cm: (8,15,17)
+48 cm: (12,16,20)
+
+In contrast, some lengths of wire, like 20 cm, cannot be bent to form an integer sided
+right angle triangle, and other lengths allow more than one solution to be found; for
+example, using 120 cm it is possible to form exactly three different integer sided
+right angle triangles.
+
+120 cm: (30,40,50), (20,48,52), (24,45,51)
+
+Given that L is the length of the wire, for how many values of L ≤ 1,500,000 can
+exactly one integer sided right angle triangle be formed?
+"""
+
+from collections import defaultdict
+from math import gcd
+from typing import DefaultDict
+
+
+def solution(limit: int = 1500000) -> int:
+    """
+    Return the number of values of L <= limit such that a wire of length L can be
+    formmed into an integer sided right angle triangle in exactly one way.
+    >>> solution(50)
+    6
+    >>> solution(1000)
+    112
+    """
+    freqs: DefaultDict = defaultdict(int)
+    m = 2
```

Comment:
Please choose descriptive variable names than `m`, `n`, `v`
If `perim` means perimeter then choose the latter.